pyCGM_Single/pyCGM.py

```diff
@@ -254,31 +270,37 @@ def hipJointCenter(frame,pel_origin,pel_x,pel_y,pel_z,vsk=None):
 
     return hip_JC
 
-def hipAxisCenter(l_hip_jc,r_hip_jc,pelvis_axis):
-    """Calculate the hip joint axis function.
 
-    Takes in a hip joint center of x,y,z positions as well as an index.
-    and takes the hip joint center and pelvis origin/axis from previous functions.
-    Calculates the hip axis and returns hip joint origin and axis.
+def hipAxisCenter(l_hip_jc, r_hip_jc, pelvis_axis):
+    """Calculate the hip center axis and hip axis.
+
+    Takes in the left and right hip joint center of x,y,z positions and pelvis
+    origin and axis, and calculates and returns the hip center and axis.
 
-    Hip center axis: Computed by taking the mean at each x,y,z axis of the left and right hip joint center.
-    Hip axis: Computed by getting the summation of the pelvis and hip center axes.
+    Hip center axis: Midpoint of left and right hip joint centers.
+
+    Hip axis: pelvis axis added back to the midpoint of left and right hip
```

Comment:
'Hip axis: sets the pelvis orientation to the hip center axis (i.e. midpoint of left and right hip joint centers)'